apps.plugin for windows (#18594)
netdata/netdata: src/collectors/apps.plugin/README.md

To achieve this task, it iterates through the whole process tree, collecting resource usage information
for every process found running.

Unlike traditional process monitoring tools (like `top`), `apps.plugin` is able to account the resource
utilization of exit processes. Their utilization is accounted at their currently running parents.
So, `apps.plugin` is able to measure the resources used by shell scripts and other processes
that fork/spawn other short-lived processes hundreds or even thousands of times per second.

## Charts sections

To provide more valuable insights, apps.plugin aggregates individual processes in several ways.
Each type of aggregation is presented as a different section on the dashboard.

### Custom Process Groups (Apps)

In this section, apps.plugin summarizes the resources consumed by all processes, grouped based

> on the groups provided in `/etc/netdata/apps_groups.conf` (to edit it on your system run
> `/etc/netdata/edit-config apps_groups.conf`).

Reviewer: ```suggestion
on the groups provided in `/etc/netdata/apps_groups.conf`. You can edit this file using our [`edit-config`](docs/netdata-agent/configuration/README.md) script.
```

Let's not re-introduce configuring here, as giving instructions on what to run will not work for everyone (I got the Agent installed at opt/). Let's point them to the configuration doc (which I will make some edits to make it more clear to use the `edit-config` script)
